app/scripts/views/settings.js

```diff
@@ -206,22 +204,8 @@ function ($, modal, Cocktail, Session, BaseView, AvatarMixin,
         })
         .fin(function () {
           self.logViewEvent('signout.success');
-          // Unset uid otherwise it will henceforth be impossible
-          // to sign in to different accounts inside this tab.
-          self.relier.unset('uid');
-          self.relier.unset('email');
-          self.relier.unset('preVerifyToken');
           self.user.clearSignedInAccount();
-          // The user has manually signed out, a pretty strong indicator
-          // the user does not want any of their information pre-filled
-          // on the signin page. Clear any remaining formPrefill info
-          // to ensure their data isn't sticking around in memory.
-          self._formPrefill.clear();
-          Session.clear();
-          self.navigate('signin', {
-            clearQueryParams: true,
-            success: t('Signed out successfully')
-          });
+          self._clearSessionAndNavigateToSignIn();
```

Comment:
We have been migrating to exposing functions provided by mixins as public, same with those that a super class exposes to a sub class. Since this function is provided by a mixin, can you make it public? Not a terribly great convention, but not the worst either.